dotcom-rendering/src/components/TextBlockComponent.tsx

```diff
@@ -303,6 +305,9 @@ const buildElementTree =
 			case 'S':
 			case 'I':
 			case 'VAR':
+			case 'STRIKE':
```

Comment:
This element is deprecated[^1]. I think we probably want to render it as `<s>`[^2]?

To do this we can create a separate case for it:

```tsx
switch (node.nodeName) {
  // The other cases...
  case 'STRIKE':
    return <s css={styles(format)}>{children}</s>;
}
```

[^1]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strike
[^2]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/s